.env

```diff
@@ -0,0 +1 @@
+NODE_ENV=development;
```

Comment:
".env" file should never be under git control, please add it onto ".gitignore" file =)
https://www.w3schools.com/git/git_ignore.asp?remote=github

Reply:
Done!